src/cli.c

```diff
@@ -614,106 +638,126 @@ main(int argc, char** argv)
       }
    }
 
+   if (compress_method == MANAGEMENT_COMPRESSION_NONE)
```

Comment:
Why ?

Reply:
We handle the default compression method at this point. If none is specified, we use the default method to compress the data. By managing it here, we avoid needing to retrieve the default method later. Since the `config` variable is already available in `main()`, it’s more convenient to get the default method here.

Reply:
`none` is default - so no `-C` means no compression

Reply:
If "none" means no compression, why do we provide this option?

Reply:
we can limit user can only input
`-C gz`
`-C gz`
`-C gz`

Reply:
Yes, and `-C none` because it is the default.

If people want to use a protocol sniffer, then they shouldn't have to deal with compression and encryption on the wire